testdata/supportiety/executable.dyml

```diff
@@ -0,0 +1,44 @@
+#!{
+
+    @@name="supportiety-server"
+    executable {
+
+        services {
+            "server.http"
+            "server.manage"
+            "server.persistence"
+            "server.query"
+        }
+
+        generator go {
+
+            package "example.com/golangee/supportiety"
+
+            lib {
+                name "encoding/json"
+                short "json"
+                version "1.17"
+                type "core"
+            }
+            lib {
+                name "encoding/xml"
+                short "xml"
+                version "1.17"
+                type "core"
+            }
+            lib {
+                name "net.http"
+                short "http"
+                version "1.*"
+                type "io"
```

Comment:
wofür ist der `type`? Die Version muss für Go wahrscheinlich immer der "v1.5.2" bzw "v0.0.0-20170915032832-14c0d48ead0c/go.mod h1:Nq..." entsprechen

Reply:
Bei der Version habe ich mir einfach für das Beispiel was ausgedacht, aber hier sollte im Bestfall eine gepinnte Version stehen.

Der Type ist da, um das Konzept der Trennung von core und io auch in den Abhängigkeiten weiter umzusetzen, wie auch [für die Services beschrieben](https://github.com/haselkern/architecture/blob/206e262f02612a22e26fa6a4c758ed13374ffb6c/docs/README.md). core-services sollen nur core-dependencies haben, damit sie die Daten verarbeiten. Nur io-services sollen io-dependencies haben, damit kein core-service anfängt z.B. Get-Requests zu starten.